Source/MediaInfo/Multiple/File_Mk.cpp

```diff
@@ -662,7 +662,16 @@ const char* Mpegv_colour_primaries(int8u colour_primaries);
 const char* Mpegv_transfer_characteristics(int8u transfer_characteristics);
 const char* Mpegv_matrix_coefficients(int8u matrix_coefficients);
 const char* Mpegv_matrix_coefficients_ColorSpace(int8u matrix_coefficients);
-extern const char* Avc_video_full_range[];
+
+const char* Mk_Video_Colour_Range(int8u range)
+{
+    switch (range)
+    {
+        case 1: return "Limited";

```

Comment:
Is this terminology used across mediainfo,or is there a value to using the Matroska 0: Unspecified, 1: Broadcast Range, 2: Full range (no clipping):

Reply:
*Matroska vocabulary I meant to say

Reply:
I use the first vocabulary I met, the MPEG one. I try to have the same vocabulary whatever is the source so I don't use Matroska termilogy (despite the fact it is more precise) and use the one I already have. So same policy in MediaConch whatever is the source.